org.civicrm.bootstrapcivihr/scss/directives/horizontal-scroll-shadows.scss

```diff
@@ -0,0 +1,40 @@
+.horizontal-scroll-shadows-master {
+  overflow: hidden; // this ensures the shadows are hidden behind
+  position: relative;
+}
+
+.horizontal-scroll-shadows-wrapper {
+  overflow-x: auto;
+
+  &::before,
+  &::after {
+    box-shadow: 0 0 10px 1px rgba(48, 40, 40, 0.5);
```

Comment:
we already have a default box-shadow value coming from shoreditch, we can't use that?

Reply:
@AkA84 in Shoreditch we have `$crm-form-item-shadow: 0 3px 6px 0 rgba(48, 40, 40, 0.25);`

while I need `0 0 10px 1px rgba(48, 40, 40, 0.5);`

I don't think I can apply then variable and then just change position props.